management/analytics/analyticsController.js

```diff
@@ -0,0 +1,74 @@
+define('analyticsController', [], function() {
+
+	function analyticsController($scope, AnalyticsDataService, NotificationUtilities) {
+
+		$scope.sites = [];
+
+		$scope.loadSites = () => {
+			AnalyticsDataService.getSites().then((result) => {
+				if (result == null) {
+					NotificationUtilities.giveNotice('Failure', 'Unable to load the sites.', false);
+					return;
+				}
+
+				$scope.sites = result;
+			});
+		};
+
+		// Number of appointments scheduled
+		// Number of UNL/Wesleyan appointments scheduled
+		// Number of international appointments scheduled
+		
+
+		$scope.addDemoData = () => {
```

Comment:
what are these comments/whitespace for?

Reply:
Ah good catch--they're for nothing. Just trying to keep stuff straight when I was writing this. I'll delete these.